README.md

```diff
@@ -43,16 +43,30 @@ pip install -r requirements.txt
 python3 app.py
 ```
 
+#### Linting
+```zsh
+# Install flake8 for linting
+pip install flake8
+
+# Run flake8 from root directory
+flake8 *.py && flake8 listeners/
+
+# Install black for code formatting
+pip install black
```

Comment:
These two dependencies are already installed via lines 39/40 above:

```suggestion
# Run flake8 from root directory
flake8 *.py && flake8 listeners/
```

Reply:
ahhh yes good catch!! updating that now!! 🙌